README.md

```diff
@@ -59,8 +62,43 @@ _NB: exact path to **python3.11** may vary; locate it with: `where python` (Wind
 
 <!-- -------------------------------------------------- -->
 
-## Unit tests
-* activate your virtualenv: `workon subtitles-creator`
+## Unit tests and coverage
+* activate your virtualenv: `workon m05-mp-decaillet`
 * run unit tests: `python -m unittest discover -v`
+* run unit tests and display coverage report: `coverage run --source=src -m unittest -v  &&  coverage report -m`
+
+[GitHub actions](.github/workflows/main.yml) will enforce unit-test coverage of 100%.
+
+<!-- -------------------------------------------------- -->
+
+## Linter
+
+The python code in this project must match [autopep8](https://pypi.org/project/autopep8/) and [isort](https://pypi.org/project/isort/) linting/formatting rules.
+
+[GitHub actions](.github/workflows/main.yml) will enforce these rules.
+
+### Reformat from command line
+* activate your virtualenv: `workon m05-mp-decaillet`
+* apply autopep8 to all local files: `autopep8 --max-line-length=120 --recursive . -aaa --in-place`
+* apply isort to all local files: `isort .`
+
+### VS-code settings
+In VS-code, the linting  can be automated as follows:
+- Install extension [python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+- Edit your **User** Settings:
+  - CMD + shift + P > "Preferences: Open User Settings (JSON)"
+- Append the following to you User Settings:
+  ```json
+  "python.formatting.autopep8Args": [
+      "--max-line-length=120"
+  ],
+  "[python]": {
+      "editor.defaultFormatter": "ms-python.python",
+      "editor.formatOnSave": true,
+      "editor.codeActionsOnSave": {
+          "source.organizeImports": true,
+      },
+  },
```

Comment:
the comma here can be confusing

Reply:
VS-code uses JSONC for settings, so trailing comma is actually legal.

By having it, it makes it possible for you to copy-paste that bit of json wherever you like in the settings file, not necessarily at then (this is useful when you use vs-code for more than 1 programming language, as you can keep settings grouped)

So it's not there by accident, and I'd like to keep id ;)